src/models/nebula36-1/assembly.md

```diff
@@ -0,0 +1,399 @@
+
+![GPU brace](./img/gpu-brace.webp)
+
+The fan splitter on the GPU brace is intended for fans mounted on the GPU brace, and should not be used for the CPU shroud fans or the bottom intake fan.
+
+- **Tools required:** Cross-head (Phillips) screwdriver  
+
+### Steps to remove the GPU brace:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case).
+2. Unscrew the four screws holding the GPU brace in place (two on the front of the case, two on the back.)
+
+![GPU brace screws](./img/gpu-brace-screws.webp)
+
+3. Pull the GPU brace out of the chassis.
+
+## Removing the CPU shroud:
+
+The CPU shroud supports one or more of the CPU fans and guides airflow from the side of the case through the CPU radiator to the back exhaust vent. The CPU shroud is custom-designed to work with the Noctua `NH-U12S` CPU cooler that's included with nebula36.
+
+![CPU shroud](./img/cpu-shroud.webp)
+
+- **Tools required:** Cross-head (Phillips) screwdriver  
+
+### Steps to remove the CPU shroud:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case).
+2. Unscrew the two side screws holding the CPU shroud onto the top crossbar.
+
+![CPU shroud top crossbar screws](./img/cpu-shroud-screws-crossbar.webp)
+
+3. Unscrew the four back screws holding the CPU shroud onto the chassis.
+    - Hold onto the shroud while removing the final screw so it doesn't fall.
+
+![CPU shroud back screws](./img/cpu-shroud-screws-back.webp)
+
+4. Pull the CPU shroud out of the chassis.
+5. When replacing the CPU shroud, ensure all fan cables are plugged into one of the fan splitter boards in the top right of the chassis.
+
+## Unpacking the included accessories:
+
+nebula36 includes the following components preinstalled:
+
+- 1x `nebula_io` power button controller board
+- 1x `MYS7523` top I/O board
+- 3x four-pin fan splitter boards
+- 1x back CPU shroud fan
+    - Be Quiet! Silent Wings 4 140mm (`BQ SIW4-14025-LF-PWM`)
+- 1x Be Quiet! 
+- 6x short velcro straps
+    - 1x on the front of the chassis
+    - 3x on the side of the chassis
+    - 1x on the bottom of the chassis
+    - 1x on the CPU shroud
+    - 1x on the GPU brace
+
+In addition, nebula36 ships with the following non-installed accessories:
+
+- 1x Screw and velcro box
+    - 15x short velcro straps
+    - 1x long velcro strap
+    - 9x motherboard standoffs
+    - 9x M3 (motherboard) screws
+    - 4x fan screws
+- 1x CPU cooler w/ fan (optional)
+    - Noctua `NH-U12S` w/ `NF-F12 PWM` fan
+- 1x GPU brace-mounted side fan (optional)
+    - Be Quiet! Silent Wings 4 120mm (`BQ SIW4-12025-MF-PWM`)
+
+### Steps to unpack the included accessories:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case), [remove the GPU brace](#removing-the-gpu-brace), and [remove the CPU shroud](removing-the-cpu-shroud).
```

Comment:
`[remove the CPU shroud](removing-the-cpu-shroud).`

Dead link, but I think it just needs a `#` at the beginning

Reply:
Fixed.